frame/dapps-staking/src/weights.rs

```diff
@@ -33,34 +34,34 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: DappsStaking RegisteredDapps (r:1 w:1)
 	// Storage: DappsStaking PreApprovalIsEnabled (r:1 w:0)
 	fn register() -> Weight {
-		(63_974_000 as Weight)
+		(38_393_000 as Weight)
```

Comment:
for the weights, can you also write the reference hardware (CPU, computer storage space, and type of your PC that ran the benchmarks) specs as a comment? I think that might help.

Ref: https://docs.substrate.io/v3/concepts/weight/

Reply:
Sure, I can add that comment just not sure why we'd need it?
You mean help reviewers with understanding the numbers?

To be fully precise, we should run benchmarks on a machine as described on the page you linked.
But we don't have access to such a machine.

We could maybe rent one - it just looks like it will be idling most of the time.

Reply:
You're right. I was just worried if lack of a benchmark spec might affect our documentations. You can ignore this comment.

Reply:
Your comment got me thinking since I was running these benchmarks on a new machine.
Before I did it on my desktop PC (which I no longer have) and performance was comparable to the Substrate reference machine (not in specs but when compared to weights generated for substrate pallets).

However, on my new machine, the results are no longer aligned.
I've benchmarked the pallet_balances and here are the results:

**Substrate benchmarks**
```
fn transfer() -> Weight {
    (70_952_000 as Weight)
	.saturating_add(T::DbWeight::get().reads(1 as Weight))
	.saturating_add(T::DbWeight::get().writes(1 as Weight))
}
```

**My benchmarks**
```
fn transfer() -> Weight {
    (47_711_000 as Weight)
	.saturating_add(T::DbWeight::get().reads(1 as Weight))
	.saturating_add(T::DbWeight::get().writes(1 as Weight))
}
```
So the constant is ~67% of the weight calculated on the proper reference machine (note that this doesn't affect
the constants weights incurred by the DB operations).
I'll talk with bLd to see if we can get something close to the reference machine for doing the benchmarks.

The incorrect weights shouldn't be a problem for now I guess (users will pay a slightly smaller fee than what
they would have payed if this was done on the reference machine). We don't have such a huge amount of traffic
for now that such imperfect measurement should cause us problems. At least that's what I think.